apps/docs/src/app/core/component-docs/message-strip/message-strip-docs.component.html

```diff
@@ -0,0 +1,42 @@
+<fd-docs-section-title [id]="'message-strip-types'" [componentName]="'messageStrip'">
+    Message Strip Types
+</fd-docs-section-title>
+<description>A number of message strip types are available. They offer preset styling for different contextual situations. Can be one of 'warning', 'success', 'information', 'error' or normal (default). </description>
+<component-example>
+    <fd-message-strip-example></fd-message-strip-example>
+</component-example>
+<code-example [exampleFiles]="messageStripBasicExample"></code-example>
+
+<separator></separator>
+
+<fd-docs-section-title [id]="'message-strip-types'" [componentName]="'messageStrip'">
+    Message Strips Without Icons
+</fd-docs-section-title>
+<description>The default message strip has no icon. Information, Success, Warning and Error message strips by default are rendered with icons. To remove them set the <code>[noIcon]</code> property to <code>true</code>. </description>
+<component-example>
+    <fd-message-strip-noicon-example></fd-message-strip-noicon-example>
+</component-example>
+<code-example [exampleFiles]="messageStripNoIconExample"></code-example>
+
+<separator></separator>
+
+<fd-docs-section-title [id]="'message-strip-types'" [componentName]="'messageStrip'">
+    Message Strips With Customizable Width
```

Comment:
@Vanessa-Cusmich Should it be:
"Message Strips With Customizable Width" => "Message Strips With Customized Width"

Reply:
@salarenko it should be customizable.